ibm_mq/datadog_checks/ibm_mq/collectors/queue_metric_collector.py

```diff
@@ -179,3 +179,38 @@ def get_pcf_queue_reset_metrics(self, queue_manager, queue_name, tags):
                     metric_full_name = '{}.queue.{}'.format(metrics.METRIC_PREFIX, metric_name)
                     metric_value = int(queue_info[pymqi_type])
                     self.send_metric(metric_type, metric_full_name, metric_value, tags)
+
+    def _collect_metadata(self, queue_manager):
+        raw_version = self._get_version(queue_manager)
+        if not raw_version:
+            self.log.debug("Version not found in stdout")
+            return
+        self.log.debug('IBM MQ version: %s', raw_version)
+        return raw_version
+        # self.set_metadata('version', raw_version)
+
+    def _get_version(self, queue_manager):
+
+        pcf = pymqi.PCFExecute(queue_manager)
+        resp = pcf.MQCMD_INQUIRE_Q_MGR()
+        try:
+            version = resp[0][2120].decode("utf-8")
```

Comment:
Is the const for `2120` available in pymqi `1.11.1` ? If so, could you upgrade the dep ?